.env.tpl

```diff
@@ -1,5 +1,5 @@
 # provide the host where the nova api is running (e.g. your virtual dev instance)
-WANDELAPI_BASE_URL="{{ .InstanceInformation.NovaApiHost }}"
```

Comment:
It's still called like that in the typescript lib, for consistency we would have to change it there as well

Reply:
Ok, we now have made it all like
- NOVA_HOST
- NOVA_PASSWORD
- NOVA_USERNAME
- NOVA_ACCESS_TOKEN

Would you change it in the ts-lib?